vsintegration/src/FSharp.Editor/FSharp.Editor.fsproj

```diff
@@ -26,6 +26,8 @@
      <EmbeddedText Include="$(MSBuildThisFileDirectory)..\..\..\src\LegacyMSBuildResolver\LegacyResolver.txt">
       <Link>LegacyResolver.txt</Link>
     </EmbeddedText>
+    <Compile Include="Common\CancellableTasks.fs" />
+    <Compile Include="common\DocumentCache.fs" />
```

Comment:
```suggestion
    <Compile Include="Common\DocumentCache.fs" />
```

Reply:
For some reason, when adding a file, it lowercases the path.